templates/sonar.properties.erb

```diff
@@ -6,8 +6,9 @@
 #
 # See also the file conf/wrapper.conf for JVM advanced settings
 #---------------------------------------------------------
-
-
+<%if @javaopts -%>
+sonar.web.javaOpts                         <%= @javaopts %>
```

Comment:
yup, its missing..... i fixed this wit an additional commit

2015-03-25 21:49 GMT+01:00 Carlos Sanchez <notifications@github.com>:

> In templates/sonar.properties.erb
> <https://github.com/maestrodev/puppet-sonarqube/pull/21#discussion_r27164424>
> :
>
> > @@ -6,8 +6,9 @@
> >  #
> >  # See also the file conf/wrapper.conf for JVM advanced settings
> >  #---------------------------------------------------------
> > -
> > -
> > +<%if @javaopts -%>
> > +sonar.web.javaOpts                         <%= @javaopts %>
>
> is it missing the : ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/maestrodev/puppet-sonarqube/pull/21/files#r27164424>.
>